daemons/gptp/common/ieee1588port.cpp

```diff
@@ -107,7 +107,7 @@ IEEE1588Port::IEEE1588Port(IEEE1588PortInit_t *portInit)
 		if (initialLogSyncInterval == LOG2_INTERVAL_INVALID)
 			initialLogSyncInterval = -5;     // 31.25 ms
 		if (initialLogPdelayReqInterval == LOG2_INTERVAL_INVALID)
-			initialLogPdelayReqInterval = -3;  // 125 ms
```

Comment:
Initial PDelayRequestInterval of 1 second is according to the Automotive Profile. There are two other commits in this pull request the impact the correct functioning of PDelay responses arriving before a first sync as well as the sync intervals of anything different than 125ms when Syntonize mode is in use. For clarity the intent of the defaults are:
Init Sync: 31.25 ms
Oper Sync: 1 second
Init PDelay Req: 1  second
Oper PDelay Req: 1 second